src/nav.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+
+import React from 'react';
+
+class Nav extends React.Component {
+    render(){
+        return(
+            <nav>
+                <ul className="mobile-hide">
+                    <img id="logo" alt="BrickHack Logo"/>
+                    <a href=""><li>GENERAL</li></a>
+                    <a href="/#about"><li>ABOUT</li></a>
```

Comment:
Why `/#about` and not `#about`?

Reply:
If no reason we can just keep as `/#about` nbd

Reply:
Using just `#about` was throwing an eslint warning, I believe it was because the actual element with #about is in another module. I'll change it back to `#about` since I decided to disable that particular warning